.travis.yml

```diff
@@ -1,3 +1,3 @@
 language: objective-c
-osx_image: xcode6.4
+osx_image: xcode7.1
```

Comment:
My bad, it's not available: http://docs.travis-ci.com/user/languages/objective-c/#Supported-OS-X-iOS-SDK-versions

You were correct in setting it to `xcode7.1` originally.